internal/controller/ols_app_postgres_assets_test.go

```diff
@@ -131,6 +131,9 @@ var _ = Describe("App postgres server assets", func() {
 				Name: PostgresDataVolume,
 				VolumeSource: corev1.VolumeSource{
 					EmptyDir: &corev1.EmptyDirVolumeSource{},
+					//					PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
```

Comment:
let's add a test case to generate the PVC,too.

Reply:
yep, this is WIP